CONTRIBUTING.md

```diff
@@ -46,13 +46,36 @@ See the [contributor's testing guide](https://github.com/kudobuilder/kudo/blob/m
 
 ## Community, Discussion, and Support
 
-Learn how to engage with the Kubernetes community on the [community page](https://kudo.dev/community/).
+Learn how to engage with the KUDO community on the [community page](https://kudo.dev/community/).
 
 You can reach the maintainers of this project at:
 
 - [Slack](https://kubernetes.slack.com/messages/kudo/)
 - [Mailing List](https://groups.google.com/d/forum/kudobuilder)
 
+## Code culture
+
+This is a set of practices we try to live by when developing KUDO. These are just defaults (soft rules). Deviations from them are possible, only has to be justified.
+
+### General guidelines
+- Master is always releasable (green CI)
+- All feature/bug-fixing work should have an open issue with a description
+- Every user-facing feature that is NOT behind a feature gate should have an integration test
```

Comment:
```suggestion
- Every user-facing feature that is NOT behind a feature gate should have an integration/e2e test
```

Reply:
I don't completely understand @zen-dog suggetion... is that integration or e2e or ?

Reply:
agree – it would be good to spell out whether this means _and_ or _or_.

Reply:
It's either-or. Thinking about pipe tasks here, which is hard to test with an integration test.